src/helpers.php

```diff
@@ -4,7 +4,7 @@
 use Akaunting\Money\Money;
 
 if (! function_exists('money')) {
-    function money(mixed $amount, string $currency = null, bool $convert = null): Money
+    function money(mixed $amount, ?string $currency = null, bool $convert = null): Money
```

Comment:
I noticed that this raises deprecation notice during `Illuminate\Foundation\ComposerScripts::postAutoloadDump`

> Deprecation Notice: money(): Implicitly marking parameter $convert as nullable is deprecated, the explicit nullable type must be used instead in /var/www/html/vendor/akaunting/laravel-money/src/helpers.php:7

```suggestion
    function money(mixed $amount, ?string $currency = null, ?bool $convert = null): Money
```

Reply:
thanks I missed that parameter